Jenkinsfile

```diff
@@ -101,23 +101,18 @@ pipeline {
             label 'docker'
           }
           environment {
-            XMOSDOC_VERSION = "v3.0.0"
+            XMOSDOC_VERSION = "pr-67"
           }
           stages {
             stage('Build Docs') {
               steps {
                 runningOn(env.NODE_NAME)
                 checkout scm
-                sh "docker pull ghcr.io/xmos/doc_builder:${XMOSDOC_VERSION}"
-                sh """docker run --user "\$(id -u):\$(id -g)" \
-                        --rm \
-                        -v ${WORKSPACE}:/build \
-                        -e EXCLUDE_PATTERNS="/build/doc/doc_excludes.txt" \
-                        -e DOXYGEN=1 -e DOXYGEN_INCLUDE=/build/doc/Doxyfile.inc \
-                        -e PDF=1 \
```

Comment:
this line has been removed, but the PDF are still generated. How do we disable the PDP generation with `xmosdoc`?

Reply:
The default builders for `xmosdoc` are `clean linkcheck html latex`. 
To run different builders you just choose that: `xmosdoc clean linkcheck html` (or just `xmosdoc html`)